api/src/db/migrations/v117_add_onboarding_nonprofit.ts

```diff
@@ -1,3 +1,4 @@
+import { randomInt } from "@shared/intHelpers";
```

Comment:
If this is going out as a hotfix I don't know that the `v117` migration should be in here.